crates/experimentation_platform/src/api/experiments/handlers.rs

```diff
@@ -316,13 +306,43 @@ pub async fn conclude(
         })?;
 
         if variant.id == winner_variant_id {
-            let context_move_req = ContextMoveReq {
-                context: experiment_context.clone(),
-            };
+            if let Some(some_experiment_context) = &experiment.context {
+                let experiment_context =
+                    some_experiment_context.as_object().ok_or_else(|| {
+                        log::error!(
+                            "could not convert the context read from DB to JSON object"
+                        );
+                        unexpected_error!(
+                            "Something went wrong, failed to conclude experiment"
+                        )
+                    })?;
+
+                let context_move_req = ContextMoveReq {
+                    context: experiment_context.clone(),
+                };
+                operations.push(ContextAction::MOVE((context_id, context_move_req)));
+            } else {
+                for (key, val) in variant.overrides {
+                    let mut create_req = Map::new();
+                    create_req.insert("value".to_string(), val);
+                    let http_client = reqwest::Client::new();
+                    let url = state.cac_host.clone()
+                        + format!("/default-config/{key}").as_str();
+                    let _ = http_client
```

Comment:
use `request` function for api calls.

Reply:
Hey we don't have that function in the experiment_platform crate
It's only present in frontend crate 
Should we add ?

Reply:
It would be better to add a similar function to backend too.

Reply:
Yeah Sure